irmaseal-core/src/meta.rs

```diff
@@ -0,0 +1,95 @@
+use super::stream::*;
+use crate::Identity;
+use super::{Error};
+use base64;
+use arrayvec::{ArrayString, ArrayVec};
+use serde::{Deserialize, Serialize, Deserializer, Serializer};
+
+/// The version of the encrypted data
+#[derive(Serialize, Deserialize, Debug, PartialEq)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum Version {
+    V1_0,
+}
+
+/// Metadata which contains the version
+#[derive(Serialize, Deserialize, Debug, PartialEq)]
+pub struct Metadata {
+    pub version: Version,
+    pub media_type: ArrayString<[u8; 255]>,
```

Comment:
Is it practical to switch to the std version of string now we dropped the requirement to use `nostd`?

Reply:
Possibly but I like this static size.